gammapy/background/tests/test_models.py

```diff
@@ -43,3 +49,100 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+class TestCubeBackgroundModel():
+
+    @remote_data
+    def test_read_bin_table(self):
+
+        # test shape of bg cube when reading a file
+        filename = '../test_datasets/background/bg_cube_model_test.fits'
```

Comment:
Done overall. Please notice, that in this case, the `get_path` line is too long (beyond 80 chars).